src/components/structures/RoomSubList.js

```diff
@@ -119,7 +136,7 @@ var RoomSubList = React.createClass({
     // The header is collapsable if it is hidden or not stuck
     // The dataset elements are added in the RoomList _initAndPositionStickyHeaders method
     isCollapsableOnClick: function() {
-        var stuck = this.refs.header.dataset.stuck;
+        var stuck = this.refs.header.refs.header.dataset.stuck;
```

Comment:
Yeah, the alternative would be a function ref, in which case you could probably pass the function through and then the ref goes straight into the function, but this isn't making things a whole lot worse, so meh.